DBM-Raids-WoTLK/RubySanctum/Halion.lua

```diff
@@ -183,6 +200,38 @@ function mod:SPELL_AURA_APPLIED(args)--We don't use spell cast success for actua
 		if self.Options.SetIconOnFireConsumption then
 			self:SetIcon(args.destName, 7)
 		end
+	elseif args:IsSpellID(74826, 74827, 74828, 74829, 74830, 74831, 74832, 74833, 74834, 74835, 74836) then -- Corporeality
+		-- 74826: 50% Corporeality
+		-- 74827: 60% Corporeality
+		-- 74828: 70% Corporeality
+		-- 74829: 80% Corporeality
+		-- 74830: 90% Corporeality
+		-- 74831: 100% Corporeality
+		-- 74832: 40% Corporeality
+		-- 74833: 30% Corporeality
+		-- 74834: 20% Corporeality
+		-- 74835: 10% Corporeality
+		-- 74836: 0% Corporeality
+		if playerInTwilight and destcId == 40142 then
+			corporeality = corporealityValueByID[spellId]
+			specWarnCorporeality:Show(corporeality)
+		elseif not playerInTwilight and destcId == 39863 then
+			corporeality = corporealityValueByID[spellId]
+			specWarnCorporeality:Show(corporeality)
+		end
+		if corporeality >= 70 and self:IsTank() then -- only voice for >= 70%, 60% is still manageable so default to the selected SA sound

```

Comment:
this code block ignores which realm a player is in

Reply:
Again didn't select block I actually selected but basically all the :Play stuff is outside the cid/realm check, so they'd just play for everyone and ignore realm and dest creature id

Reply:
Corp value is assigned based on realm+cid.
Having that value set, and all the operators being equal or less than, you would never run into the inverse corporeality value of the other realm.

Having said that, the problem is I'd have the corp cached, so it would run the play twice, one for each corporeality spellid passed. I'm thinking of re-using `previousCorporeality` that I have commented out. Do you agree?

Reply:
Check if this makes sense: [f1341a4](https://github.com/DeadlyBossMods/DBM-WotLK/pull/58/commits/f1341a48101283b9a4699c9e832fa836e8a0420e)